.github/workflows/infra-cd.yml

```diff
@@ -1,4 +1,4 @@
-name: Infra Continuous Delivery
+name: Run terraform apply on prod westeurope infrastructure
```

Comment:
If you agree, I'd use a short name because of the way GitHub UI is designed. Example `Release Infra weu`

Reply:
@gunzip suggested to explicitly explain the operation being performed (plan or apply). What do you think about Terraform Plan weu and Terraform Apply weu?

Reply:
Is there any need to specify the region?

Reply:
Currently, the workflows that execute plan and apply are different for the two regions where the infrastructure is deployed (itn and weu). I think that adding the region gives additional clarity on which region is being deployed.